doc/md/transactions.md

```diff
@@ -102,23 +102,29 @@ func WithTx(ctx context.Context, client *ent.Client, fn func(tx *ent.Tx) error)
 	if err != nil {
 		return err
 	}
+
 	defer func() {
 		if v := recover(); v != nil {
 			tx.Rollback()
 			panic(v)
 		}
 	}()
+
 	if err := fn(tx); err != nil {
 		if rerr := tx.Rollback(); rerr != nil {
-			err = errors.Wrapf(err, "rolling back transaction: %v", rerr)
+			err = fmt.Errorf("rolling back transaction: %w", rerr)
 		}
+
 		return err
 	}
+
 	if err := tx.Commit(); err != nil {
-		return errors.Wrapf(err, "committing transaction: %v", err)
+		return fmt.Errorf("committing transaction: %w", err)
 	}
+
 	return nil
 }
+
 ```
```

Comment:
Looks good, just remove the extra linebreaks please and I'll merge it. Thanks 🙏

Reply:
The extra linebreaks are suggested by https://github.com/bombsimon/wsl. But I'll remove, no prob.